### PR TITLE
CCTRI 1313/tweak documentation

### PR DESCRIPTION
**Pull request by GeorgiyII**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -10,4 +10,4 @@ install:
 
 script:
 - flake8 .
-- pytest -v tests/unit/
+- coverage run --source api/ -m pytest --verbose tests/unit/ && coverage report
```
Comment (gdavoian) [inline .travis.yml:13]: Please use a structure like here: https://github.com/CiscoSecurity/tr-05-serverless-google-safe-browsing/blob/CCTRI-1305/.travis.yml

Comment (GeorgiyII) [reply .travis.yml]: done

File: .travis.yml [hunk 2]
```diff
@@ -1,13 +1,12 @@
...
 install:
-- pip install -U -r requirements.txt
-- pip install -U -r test-requirements.txt
+- pip install --upgrade --requirement requirements.txt
+- pip install --upgrade --requirement test-requirements.txt
 
 script:
 - flake8 .
-- pytest -v tests/unit/
+- coverage run --source api/ -m pytest --verbose tests/unit/ && coverage report
```
Comment (gdavoian) [inline .travis.yml:12]: Please also fix the `New line at end of file` GitHub warning.

Comment (GeorgiyII) [reply .travis.yml]: done

File: README.md [hunk 3]
```diff
@@ -1,73 +1,211 @@
 [![Travis CI Build Status](https://travis-ci.com/CiscoSecurity/tr-05-serverless-cyberprotect.svg?branch=develop)](https://travis-ci.com/CiscoSecurity/tr-05-serverless-cyberprotect)
 
-# Threatscore Cyberprotect Relay API
+# Threatscore | Cyberprotect Relay API
```
Comment (gdavoian) [inline README.md:3]: Looks like the original term `Relay API` has been replaced with just `Relay` over time, so please remove the word `API` :)

File: README.md [hunk 4]
```diff
@@ -1,73 +1,211 @@
...
-**Note.** For convenience, each TR Relay CLI command may be prefixed with
-`env $(cat .env | xargs)` to automatically read the required environment
-variables from a `.env` file (i.e.`TR_API_CLIENT_ID`, `TR_API_CLIENT_PASSWORD`,
-`URL`) and pass them to the corresponding command.
+### Threat Response Module
 
-## Environment Variables
+Now, the only things left to do is to follow one of these URLs to navigate 
+to Threat Response in your region and create the Cyberprotect module using your Lambda's URL:
+  - US: https://visibility.amp.cisco.com/settings/modules/available/a89161ba-8d70-4ea9-a190-1453a763d84f/new
```
Comment (gdavoian) [inline README.md:133]: These extra indents aren't needed.

Comment (GeorgiyII) [reply README.md:133]: why?
without jwt user will not use a script and does not recognize these urls

Comment (gdavoian) [reply README.md:133]: There's no JWT in this integration :) That's why there's are no links to the JWT generator script.

You're just rendering an unordered list of items in Markdown, so you don't those indents, e.g.:
```
- US: https://visibility.amp.cisco.com/settings/modules/available/a89161ba-8d70-4ea9-a190-1453a763d84f/new
- EU: ...
- APJC: ...
```

Comment (GeorgiyII) [reply README.md]: done

**Pull request merged**